src/main/kotlin/org/rust/lang/core/resolve/KnownItems.kt

```diff
@@ -109,7 +109,7 @@ class KnownItems(
     val Debug: RsTraitItem? get() = findLangItem("debug_trait")
     val Box: RsStructOrEnumItemElement? get() = findLangItem("owned_box", "alloc")
     val Rc: RsStructOrEnumItemElement? get() = findItem("alloc::rc::Rc")
-    val Arc: RsStructOrEnumItemElement? get() = findItem("alloc::sync::Arc")
+    val Arc: RsStructOrEnumItemElement? get() = findItem("std::sync::Arc")
```

Comment:
Hmm, does some test failed on this? It can be a problem in a project with `#![no_std]` and `extern crate alloc;`

Reply:
Yeah, `findItem("alloc::sync::Arc") ?: findItem("std::arc::Arc")` looks ok